docs/cli.rst

```diff
@@ -57,43 +65,43 @@ List available versions of Qt, targets, extensions, modules, and architectures.
     Extension of packages to list
     {wasm,src_doc_examples,preview,wasm_preview,x86_64,x86,armv7,arm64_v8a}
 
-    Use the `--extensions` flag to list all relevant options for a host/target.
-    Incompatible with the `--extensions` flag, but may be combined with any other flag.
+    Use the ``--extensions`` flag to list all relevant options for a host/target.
+    Incompatible with the ``--extensions`` flag, but may be combined with any other flag.
 
 .. option:: --spec <Specification>
 
     Print versions of Qt within a version specification, as explained here:
     https://python-semanticversion.readthedocs.io/en/latest/reference.html#semantic_version.SimpleSpec
```

Comment:
It can be a reference and link syntax.

Reply:
We can do that if you want. I think it's a little silly to link to something that's just a few lines down though, and that maybe it would make more sense to simply reorder the options so that `--extension` appears right next to `--extensions`. I will try it with the link first though.

Reply:
I will try turning every mention of a flag into a link to the docs for that flag. I think it would be inconsistent to do this for just one flag and not the others.

Reply:
I've just comment about URL ` https://python-semanticversion.readthedocs.io/en/latest/reference.html#semantic_version.SimpleSpec`

Reply:
Oops, I misunderstood! Thanks for the clarification; this should be an easy fix.

I will revert ddc3606 as well.